harness/determined/cli/agent.py

```diff
@@ -19,25 +20,23 @@ def local_id(address: str) -> str:
 
 @authentication.required
 def list_agents(args: argparse.Namespace) -> None:
-    r = api.get(args.master, "agents")
-
-    agents = r.json()
+    resp = bindings.get_GetAgents(setup_session(args))
     agents = [
         OrderedDict(
             [
-                ("id", local_id(agent_id)),
-                ("version", agent["version"]),
-                ("registered_time", render.format_time(agent["registered_time"])),
-                ("num_slots", len(agent["slots"])),
-                ("num_containers", agent["num_containers"]),
-                ("resource_pool", agent["resource_pool"]),
-                ("enabled", agent.get("enabled")),
-                ("draining", agent.get("draining")),
-                ("label", agent["label"]),
-                ("addresses", ", ".join(agent["addresses"])),
+                ("id", local_id(a.id)),
+                ("version", a.version),
+                ("registered_time", render.format_time(a.registeredTime)),
+                ("num_slots", len(a.slots)),
+                ("num_containers", len(a.containers)),
+                ("resource_pool", a.resourcePool),
+                ("enabled", a.enabled),
+                ("draining", a.draining),
+                ("label", a.label),
+                ("addresses", ", ".join(a.addresses)),
             ]
         )
-        for agent_id, agent in sorted(agents.items())
+        for a in sorted(resp.agents, key=lambda x: x.id)
```

Comment:
there's a python lambda police that likes to enforce

```suggestion
        from operator import attrgetter
        for a in sorted(resp.agents, key=attrgetter("id"))
```

Reply:
neat